go-controller/pkg/util/gateway-init.go

```diff
@@ -77,8 +80,8 @@ func GetDefaultGatewayRouterIP() (string, net.IP, error) {
 	return routers[0].name, routers[0].ip, nil
 }
 
-// getGatewayLoadBalancers find TCP UDP load-balancers from gateway router.
-func getGatewayLoadBalancers(gatewayRouter string) (string, string, error) {
+// GetGatewayLoadBalancers find TCP UDP load-balancers from gateway router.
+func GetGatewayLoadBalancers(gatewayRouter string) (string, string, error) {
 	lbTCP, stderr, err := RunOVNNbctl("--data=bare", "--no-heading",
```

Comment:
This exported function is not used outside of this package, perhaps you should rever the change above?

Reply:
I think I needed this until I found out there was another function that does pretty much the same thing in ovn package. Will revert.